app/src/main/res/layout/continue_interaction_item.xml

```diff
@@ -46,14 +50,19 @@
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent" />
 
-    <Button
-      android:id="@+id/continue_button"
+    <org.oppia.android.app.customview.ContinueButtonView
```

Comment:
Shouldn't the navigation button layout & view model also be updated to use this custom view?

Reply:
done